File: ServerTraveling/DomainService/Models/Characteristic.cs
using DomainService.Helpers.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DomainService.Models
{
    public class Characteristic
    {
        public PriceCharacteristics PriceCharacteristic { get; set; }
        public LocalityCharacteristics LocalityCharacteristic { get; set; }
        public AbroadCharacteristics AbroadCharacteristic { get; set; }
        public CuisineCharacteristics CuisineCharacteristic { get; set; }
        public TransportCharacteristics TransportCharacteristics { get; set; }
        public AccessibilityCharacteristics AccessibilityCharacteristic { get; set; }
        public MonumentsCharacteristics MonumentsCharacteristic { get; set; }
    }
}

File: client-traveling/src/Models/Characteristic.ts
import { AbroadCharacteristics, AccessibilityCharacteristics, CuisineCharacteristics, LocalityCharacteristics, MonumentsCharacteristics, PriceCharacteristics, TransportCharacteristics } from "../Enums/CharacteristicsEnums"

export interface Characteristic
{
    priceCharacteristic: PriceCharacteristics,
    localityCharacteristic: LocalityCharacteristics,
    abroadCharacteristic : AbroadCharacteristics,
    cuisineCharacteristic : CuisineCharacteristics
    transportCharacteristics : TransportCharacteristics
    accessibilityCharacteristic : AccessibilityCharacteristics
    monumentsCharacteristic : MonumentsCharacteristics
}